app/grandchallenge/cases/models.py

```diff
@@ -358,6 +361,67 @@ def shape(self) -> List[int]:
             result.append(color_components)
         return result
 
+    @property
+    def spacing(self) -> List[float]:
+        """
+        Return the voxel spacing (or size if spacing is nonexistent) of the image.
+
+        Returns
+        -------
+            The voxel spacing in mm in SimpleITK ordering [x, y, (z)]
```

Comment:
Why does this return the ordering in a different ordering to shape? This will cause issues down the road, we should stick to 1 ordering for arrays. This is why we explicitally set width, height and depth.

Reply:
I did this so it the order is equivalent to the output of SimpleITK's `Image.GetSpacing()` but indeed it is probably better to keep it consistent with shape and change it to NumPy ordering.